src/test/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceServiceSpec.scala

```diff
@@ -151,6 +152,101 @@ class WorkspaceServiceSpec extends FreeSpec with ScalaFutures with ScalatestRout
       }
     }
 
+    "when calling POST on the workspaces/*/*/importEntities path" - {
+
+      "should 400 Bad Request if the first column header of the TSV is an unknown entity type" in {
+        val path = FireCloudConfig.Workspace.importEntitiesPathFromWorkspace(
+          MockWorkspaceServer.mockValidWorkspace.namespace.get,
+          MockWorkspaceServer.mockValidWorkspace.name.get)
+        (Post(path, MockTSVFormData.unknownFirstColumnHeader)
+          ~> Cookie(HttpCookie("iPlanetDirectoryPro", token))
+          ~> sealRoute(routes)) ~> check {
+          status should equal(BadRequest)
+        }
+      }
+
+      //IGNORED - a collection-type TSV missing members header is currently indistinguishable from an entity-update TSV.
```

Comment:
Can we add `TODO` in here so it will show up as such in the IDE? Also, are we adding backlog stories to address this kind of future intention?